examples/word-count/src/word_count.clj

```diff
@@ -249,19 +243,22 @@
   ;; Evaluate the form:
   (reset)
 
+  ;; Evaluate the form:
+  (require '[clojure.java.io :as io])
+
   ;; Evaluate the form:
   (let [text-input (slurp (io/resource "metamorphosis.txt"))
         values (str/split text-input #"\n")]
     (doseq [v values]
       (publish (:input topic-metadata) nil v)
-      (info v))
-    (info "The End"))
+      (println v))
```

Comment:
It might be nice not to print every line, but some other (smaller) progress indicator. I use a fast terminal and it took a while - in an emacs buffer I imagine its a bit painful ...